### PR TITLE
[UI][fix] - test fix for gh-5100

### PR DESCRIPTION
**Pull request by mkralik3**
fix for UI
//test: `@integration-metrics`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/summary/MetricsSteps.java [hunk 1]
```diff
@@ -49,6 +50,10 @@ public void checkNumberOfTotalError(int numberOfErrors) {
     public void checkLastProcessed() throws ParseException {
         refresh();
         String dateLabel = metricsTab.getLastProcessed();
+        if(metricsTab.getNumberOfTotalMessages()==0){
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/summary/MetricsSteps.java]: change style to: `if (metricsTab.getNumberOfTotalMessages() == 0) {`

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**